scripts/ci-docker-images/builder-x86_64-unknown-linux-gnu/Dockerfile

```diff
@@ -0,0 +1,14 @@
+# The official Rust image based on Debian 8 Jessie.
```

Comment:
This file seems much much simpler 👍. Are we statically linking `leveldb` and `rdkafka`?

Reply:
> Are we statically linking leveldb and rdkafka?

Yes!